README.md

```diff
@@ -5,9 +5,9 @@ This [Spring-Boot starter](http://docs.spring.io/spring-boot/docs/current-SNAPSH
 
 This client uses the [AutoConfiguration](http://docs.spring.io/spring-boot/docs/current-SNAPSHOT/reference/htmlsingle/#using-boot-auto-configuration "Spring Boot docu") feature of Spring Boot to register service and controller beans in the application context.
 
-The main service that is used is a registrar that registeres the application at the spring-boot-admin application by periodically calling a REST-API to check or perform the registration of itself.
+The main service that is used is a registrar that registers the application at the spring-boot-admin application periodically calling a REST-API to check or perform the registration of itself.
```

Comment:
Thanks for the fixes! The sentence is still a bit confusing though. Does it mean that the main service periodically callss a REST-API to register the application at spring-boot-admin? If so, then the *by* is useful IMO. The rest LGTM!